.circleci/doc-docker/Dockerfile

```diff
@@ -1,9 +1,17 @@
 FROM drmike/r-bookdown
 
+RUN Rscript -e 'install.packages("tufte")'
```

Comment:
Best practice to add these after the other install steps so that they can remain cached